input/groups/back.json

```diff
@@ -93,16 +100,8 @@
         2344,
         9051
       ],
-      "Standalone: World Boss": [
-        2358,
-        6621
```

Comment:
Where did this go? I don't see it elsewhere in the file - I assume it was accidentally dropped?

Reply:
Just checked, it's on "Standalone: Other", right below. the "Standalone: World Boss" group became redundant with the new event filter, and it doesn't contain enough data to warrant the space. You'll see I removed other minor standalone groups in future pull requests as well.

The term "World Boss" is kinda ambiguous as well, not defined enough for my taste.

Reply:
Sorry, the line quote wasn't very clear. I specifically meant back 6621 which disappeared from the groupings entirely.

Reply:
Oh I see, it was double-categorised before.